starter-project-backend-g32/src/models/UserProfile.ts

```diff
@@ -0,0 +1,24 @@
+import { model, Schema } from "mongoose";
+
```

Comment:
Use Interface to define the structure of the document, and define schema type based on the interface